netplan_cli/cli/state.py

```diff
@@ -193,26 +224,9 @@ def down(self) -> bool:
 
     @property
     def type(self) -> str:
-        match = dict({
-            'bond': 'bond',
-            'bridge': 'bridge',
-            'ether': 'ethernet',
-            'ipgre': 'tunnel',
-            'ip6gre': 'tunnel',
-            'loopback': 'ethernet',
-            'sit': 'tunnel',
-            'tunnel': 'tunnel',
-            'tunnel6': 'tunnel',
-            'wireguard': 'tunnel',
-            'wlan': 'wifi',
-            'wwan': 'modem',
-            'vlan': 'vlan',
-            'vrf': 'vrf',
-            'vxlan': 'tunnel',
-            })
         nd_type = self.nd.get('Type') if self.nd else None
-        if nd_type in match:
-            return match[nd_type]
+        if nd_type := DEVICE_TYPES.get(nd_type, nd_type):
+            return nd_type
```

Comment:
I think this doesn't fully match up (I guess I was confused earlier). We're now returning the old `nd_type` value, but we actually want to return `None` if it's not found, but still log the old `nd_type` value in the warning.

Maybe we should just keep the original logic here, without the `:=` shortcut?